### DISCUSSION TITLE
部署出现一大堆错误

### DISCUSSION THREAD

**Discussion by liupeng176** [RapidAI/RapidRAG#20] [Q&A]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

很多依赖错误，建议作者自己从一个新的操作系统安装走一遍流程

### Expected behavior

很多依赖错误，建议作者自己从一个新的操作系统安装走一遍流程

### Screenshots

1

### Environment

```markdown
- OS:1
- Browser:1
- Browser Version:1
```

**Comment by SWHL** [comment]
建议给出一些遇到的具体问题，我这里手头没有新的机器部署。

**Comment by liupeng176** [comment]
哎，有点多  我的是amd的显卡，不知道是不是不兼容还是怎么？
直接搞个虚拟机 vm的，搞完删除掉就好了

**Comment by SWHL** [comment]
AMD的显卡，单独部署大模型可以跑通吗？

**Comment by qiruizheng** [comment]
能否支持docker部署
